plugins/git-extras/git-extras.plugin.zsh

```diff
@@ -227,59 +270,62 @@ _git-refactor() {
 }
 
 
-_git-bug() {
-    local curcontext=$curcontext state line ret=1
-    declare -A opt_args
-
-    _arguments -C \
-        ': :->command' \
-        '*:: :->option-or-argument' && ret=0
+_git-squash() {
+    _arguments \
+        ':branch-name:__git_branch_names'
+}
 
-    case $state in
-        (command)
-            declare -a commands
-            commands=(
-                'finish:merge bug into the current branch'
-            )
-            _describe -t commands command commands && ret=0
-            ;;
-        (option-or-argument)
-            curcontext=${curcontext%:*}-$line[1]:
-            case $line[1] in
-                (finish)
-                    _arguments -C \
-                        ':branch-name:__git_bug_branch_names'
-                    ;;
-            esac
-    esac
+_git-summary() {
+    _arguments '--line[summarize with lines other than commits]'
+    __git_commits
 }
 
 
+_git-undo(){
+    _arguments  -C \
+        '(--soft -s)'{--soft,-s}'[only rolls back the commit but changes remain un-staged]' \
+        '(--hard -h)'{--hard,-h}'[wipes your commit(s)]'
+}
+
 zstyle ':completion:*:*:git:*' user-commands \
+    alias:'define, search and show aliases' \
+    archive-file:'export the current HEAD of the git repository to a archive' \
+    back:'undo and stage latest commits' \
+    bug:'create a bug branch' \
     changelog:'populate changelog file with commits since the previous tag' \
+    commits-since:'list commits since a given date' \
     contrib:'display author contributions' \
     count:'count commits' \
+    create-branch:'create local and remote branch' \
     delete-branch:'delete local and remote branch' \
+    delete-merged-brancees:'delete merged branches'\
```

Comment:
It looks there is a typo with `delete-merged-brancees`. I think it must be `delete-merged-branches`.